src/test/java/com/rackspace/salus/monitor_management/services/TestMonitorServiceTest.java

```diff
@@ -376,8 +375,8 @@ private void commonPerformTestMonitorOnResource_remote_normal(List<String> monit
         .thenReturn(Optional.of(resource));
 
     // just return the zones given
-    when(monitorManagement.determineMonitoringZones(any(), any()))
-        .then(invocationOnMock -> invocationOnMock.getArgument(1));
+    when(monitorManagement.determineMonitoringZones(any(), (String) any()))
```

Comment:
perhaps
```suggestion
    when(monitorManagement.determineMonitoringZones(any(), anyString()))
```

and if that works there's a couple more below

Reply:
When trying different options here I had a feeling this comment would come.  `anyString()` doesn't work when `null` is passed to it, so you have to cast to a string instead.

Reply:
and to be clearer, I only cast to a string because there are two options for this methid:

`determineMonitoringZones(List<String> zones, String region)`
`determineMonitoringZones(Monitor monitor, ResourceDTO resource)`

The alternative would be casting the first parameter to a List.